File: UTC Time Zone/UTC_time_zone.rb
#
#   Checks if time zone is Coordinated Universal Time (UTC), Ruby
#   Module written by Brandon Arvanaghi
#   Website: arvanaghi.com 
#   Twitter: @arvanaghi
#

if Time.now.getlocal.zone == "Coordinated Universal Time"
	puts "The time zone is Coordinated Universal Time (UTC), do not proceed."
else
	puts "The time zone is #{Time.now.getlocal.zone}. Proceed!"
end

File: UTC Time Zone/UTC_time_zone.go
/*
    Checks if time zone is Coordinated Universal Time (UTC), Go
    Module written by Brandon Arvanaghi 
    Website: arvanaghi.com 
    Twitter: @arvanaghi
*/

package main

import (
	"fmt"
	"time"
)

func main() {

    zoneAbbrv, offsetFromUTC := time.Now().Zone()
    if offsetFromUTC == 0 {
    	fmt.Println("The time zone is Coordinated Universal Time (UTC), do not proceed.")
    } else {
    	fmt.Println("The time zone is " + zoneAbbrv + ". Proceed!")
    }

}